src/components/ToppingsSelect/index.jsx

```diff
@@ -1,16 +1,37 @@
 import React, { useState } from "react";
 import Topping from "../Topping";
-import './style.css';
+import "./style.css";
 
 const ToppingsSelect = ({ toppings }) => {
+  const [checked, setChecked] = useState(toppings);
+
+  const handleCheck = (index) => {
+    const newToppings = [...toppings];
+    newToppings[index].selected = !checked[index].selected;
+    setChecked(newToppings);
+  };
+
+  const selectedToppings = checked.filter((topping) => topping.selected);
+
+  let totalPrice = 0;
+
+  selectedToppings.forEach((topping) => (totalPrice += topping.price));
```

Comment:
troufneš si tohle udělat pomocí .reduce? ;) https://stackoverflow.com/questions/46680151/using-reduce-to-get-sum-of-all-numbers-in-an-array-in-javascript